apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -318,6 +318,14 @@ const Ticket = new GQLListSchema('Ticket', {
             schemaDoc: 'In the case of remote system sync, you can store some extra analytics. Examples: email, name, phone, ...',
             type: Json,
         },
+        lastResidentCommentAt: {
```

Comment:
It's a little dirty. I don't think that we need to separate this field. We just need to have one field for that.